docs/api/password.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Out of curiosity: what is the benefit of having a pure vim script to generate password when there are already very well tested core utils which are doing the same and just need to be shelled out. It is unlikely we will be able to reach the same level of testing projects like https://linux.die.net/man/1/pwgen have.